static/js/app.js

```diff
@@ -279,26 +279,56 @@ function addEventListeners() {
 	});
 }
 
+function hasParent(el, selector) {
```

Comment:
nit: rename this function parameter from `selector` to `class` or something more accurate?